src/pages/index.js

```diff
@@ -46,9 +91,50 @@ const imagePopup = new PopupWithImage(".popup-photo");
 imagePopup.setEventListeners();
 
 /* ------------------------------- Card Popup ------------------------------- */
+
+function handleDeleteCard(card) {
+  confirmationPopup.open(card);
+  confirmationPopup.setSubmitAction(() => {
+    api
+      .deleteCard(card._id)
+      .then(() => {
+        card._handleRemoveCard(this);
```

Comment:
`_handleRemoveCard`

Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.

Reply:
`this` is not needed here. It has no effect


It will be:

```js

        card.handleRemoveCard();

```